src/main/resources/mimoto-issuers-config.json

```diff
@@ -20,23 +62,68 @@
         {
           "name": "e-Signet",
           "logo": {
-            "url": "https://esignet.dev1.mosip.net/logo.png",
+            "url": "https://api.dev1.mosip.net/inji/digit-icon.png",
```

Comment:
url has to be "https://esignet.dev1.mosip.net/logo.png" as it's eSignet logo not digit icon
Need to update for all languages

Reply:
updated

Reply:
updated